provider/storage_benchmark.py

```diff
@@ -0,0 +1,293 @@
+"""
+Module for providing storage benchmark tools for testing block device.
+"""
+
+import logging
+import os
+import re
+import time
+
+from virttest import utils_misc
+from virttest import postprocess_iozone
+from platform import machine
+
+
+def symbol_link_force(test, session, src, dst, timeout=60):
+    """
+    Create a soft symbol link inside guest.
+
+    :param test: kvm test object.
+    :param session: session connected guest.
+    :param src: source file.
+    :param dst: destination file.
+    :param timeout: timeout for creating link.
+    """
+    symbol_link_cmd = 'ln -s %s %s' % (src, dst)
```

Comment:
why not using `-f` to force create if destination exists.

Reply:
You are right, I forget the '-f' option, thank your comment, I will update it.

Reply:
Updated it:

>     symbol_link_cmd = 'ln -s -f %s %s' % (src, dst)
>     status, output = session.cmd_status_output(symbol_link_cmd,
>                                                timeout=timeout)
>     if status:
>         test.error('Failed to create symbol link. %s' % output)